crates/ruff_server/src/server/api/requests/execute_command.rs

```diff
@@ -40,6 +41,15 @@ impl super::SyncRequestHandler for ExecuteCommand {
         let command =
             Command::from_str(&params.command).with_failure_code(ErrorCode::InvalidParams)?;
 
+        if let Command::Debug = command {
+            let output = debug_information(session);
+            #[allow(clippy::print_stderr)]
+            {
+                eprintln!("Debug information:\n{output}");
+            }
+            return Ok(None);
```

Comment:
You might already be aware of this but printing to stderr is always displayed with Error log level in Neovim. We don't need to solve this here but I'm not sure if the server should print anything to stderr.

Reply:
Pyright sends this information as a log message at INFO level: https://github.com/microsoft/pyright/blob/2b0a1d92dda37fb80420e80d1d3933d9287be21f/packages/pyright-internal/src/commands/dumpFileDebugInfoCommand.ts#L239-L241

Reply:
It's a bit unreadable in Neovim unless the user would manually format it.
```
[ERROR][2024-06-11 12:14:00] .../vim/lsp/rpc.lua:770	"rpc"	"/Users/dhruv/work/astral/ruff/target/debug/ruff"	"stderr"	'Debug information:\npath = /Users/dhruv/work/astral/ruff/target/debug/ruff\nversion = 0.4.8\nencoding = UTF16\nopen_document_count = 1\nactive_workspace_count = 1\nconfiguration_files = ["/Users/dhruv/playground/ruff/pyproject.toml", "/Users/dhruv/playground/ruff/repro/ruff.toml"]\ncapabilities.code_action_deferred_edit_resolution = true\ncapabilities.apply_edit = true\ncapabilities.document_changes = false\ncapabilities.workspace_refresh = true\ncapabilities.pull_diagnostics = true\n\n    \n'
```

Reply:
Sure, we can make this a log message. This means it won't appear on Helix, but Helix doesn't support LSP commands anyway, so it should be fine.

Reply:
Thank you for doing that! Now, it's much better:
```
2024-06-13 07:58:53 [INFO] executable = /Users/dhruv/work/astral/ruff/target/debug/ruff
version = 0.4.8
encoding = UTF16
open_document_count = 1
active_workspace_count = 1
configuration_files = ["/Users/dhruv/playground/ruff/pyproject.toml", "/Users/dhruv/playground/ruff/repro/ruff.toml"]
capabilities.code_action_deferred_edit_resolution = true
capabilities.apply_edit = true
capabilities.document_changes = false
capabilities.workspace_refresh = true
capabilities.pull_diagnostics = true
```